src/main/java/org/fundacionjala/coding/abel/FizzBuzz.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.coding.abel;
+
+
+/**
+ *
+ */
+public class FizzBuzz {
+
+    private static final int FIZZ_NUM = 3;
+    private static final int BUZZ_NUM = 5;
+    private static final String FIZZBUZZ = "FizzBuzz";
+    private static final String FIZZ = "Fizz";
+    private static final String BUZZ = "Buzz";
+    private static final int LENGTH = 100;
+
+    /**
+     * @param indicadio indicador de entrada
+     * @return value if return
+     */
+    public String fizzBuzz(int indicadio) {
+        String[] container = generated();
+        return container[indicadio - 1];
+    }
+
+
+    /**
+     * @return value og return
+     */
+    public String[] generated() {
+        String[] a = new String[LENGTH];
+        int cont = 0;
+        for (int i = 1; i <= LENGTH; i++) {
+            a[cont] = i % FIZZ_NUM == 0 && i % BUZZ_NUM == 0 ? FIZZBUZZ
+                    : i % FIZZ_NUM == 0 ? FIZZ
+                    : i % BUZZ_NUM == 0 ? BUZZ
+                    : String.valueOf(i);
+            cont++;
+        }
+        return a;
+    }
+
+    /**
+     * @param indicadio vaue of input
+     * @return value return
+     */
+    public String fizzBuzz2(int indicadio) {
+
+        String[] container = generated2();
+
+        return container[indicadio - 1];
+
+
+    }
+
+    /**
+     * @return value of return
+     */
+    public String[] generated2() {
+
+        String[] a = new String[LENGTH];
+        int cont = 0;
+        final int num1 = 53;
+        final int num2 = 35;
+        final int num3 = 0;
+
+        for (int i = 1; i <= LENGTH; i++) {
+            a[cont] = i == num1 || i == num2 ? FIZZBUZZ
+                    : i % FIZZ_NUM == num3 || String.valueOf(i).contains("3") ? FIZZ
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)